sdk/python/src/datastar_py/consts.py

```diff
@@ -1,96 +1,102 @@
 # This is auto-generated by Datastar. DO NOT EDIT.
 from enum import StrEnum
 
-#region The mode in which a fragment is merged into the DOM.
+
+# region Enums
+
+
+# region The mode in which a fragment is merged into the DOM.
 class FragmentMergeMode(StrEnum):
     # Morphs the fragment into the existing element using idiomorph.
-    FragmentMergeModeMorph = "morph"
+    MORPH = "morph"
```

Comment:
Please revert to the convention used by all SDKs, in this case it should be `FRAGMENT_MERGE_MODE_MORPH`.

Reply:
Even though it's used with the namespace already, i.e. `FragmentMergeMode.MORPH`? Having it doubled feels redundant and very long. `FragmentMergeMode.FRAGMENT_MERGE_MODE_MORPH`

Reply:
I had a peek at a few other SDKs and it seems they follow this convention.
https://github.com/starfederation/datastar/blob/db376dfbe4d100ea997c932f1301fbe72e08dfe8/sdk/java/core/src/main/java/starfederation/datastar/enums/FragmentMergeMode.java#L6-L9
https://github.com/starfederation/datastar/blob/db376dfbe4d100ea997c932f1301fbe72e08dfe8/sdk/dotnet/src/Consts.fs#L66-L69

Reply:
Yeah you’re right, better to avoid duplication.